### PR TITLE
Test run and download methods

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: resdk/resources/utils.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def iterate_fields(fields, schema):
 
     """
     schema_dict = {val['name']: val for val in schema}
-    for field_id, properties in fields.iteritems():
+    for field_id, properties in fields.items():
```
Comment (mstajdohar) [inline resdk/resources/utils.py]: I see that tests have yield the first results. Good job!

**Comment by mstajdohar** [comment]
@JureZmrzlikar, I think this work on tests is very important. Nevertheless it will wait until the end of current milestone. It has to be reviewed well, but Domen and I are busy building the requested features.

**Comment by mstajdohar** [comment]
@JureZmrzlikar, could split the two bug fixes to a separate commit and create a separate PR? I wish to accept those tomorrow.

**Comment by dblenkus** [comment]
@JureZmrzlikar I think that tests are too long. Try to keep them as short as possible (few lines, at most one screen) and put common logic in `setUp` function. This way they will be more transparent, quicker to run (at least one test) and easier to isolate bug.

**Comment by mstajdohar** [comment]
Good point @dblenkus.

**Pull request closed without merging**